Healthy/Classes/ReusableViews/FormTextField/FormTextField.swift

```diff
@@ -0,0 +1,78 @@
+
+import UIKit
+
+class FormTextField: UIView {
+    
+    // MARK: - Outlets
+    //
+    
+    @IBOutlet weak private var contentView: UIView!
+    @IBOutlet weak private var titleLabel: UILabel!
+    @IBOutlet weak private var inputTextField: UITextField!
+    @IBOutlet weak private var errorLabel: UILabel!
+    
+    // MARK: - Properties
+    //
+    var text: ((String)->())?
```

Comment:
```suggestion
    // MARK: Properties

    private var onChange: (String) -> () =  { _ in }
```
Text is a noun, and this is a closure. I would suggest renaming it to `onChange` and add a setter method. 

Example: 
```swift
func onChange(_ onChange: @escaping (String) -> Void { 
    self.onChange = onChange
}
```